samples/gitchurn/src/gitChurnMain/kotlin/GitChurn.kt

```diff
@@ -35,11 +35,14 @@ fun main(args: Array<String>) {
 private fun calculateChurn(workDir: String, limit: Int) {
     println("Opening…")
     val repository = git.repository(workDir)
-    val map = mutableMapOf<String, Int>()
+    //val map = mutableMapOf<String, Int>()
```

Comment:
Commented code is not supposed to be kept in the repository.
Old version can always be retrieved from Git history instead.